spec/vcr_cassettes/firmware_registry/rest_api_depot_when-bad-host.yml

```diff
@@ -0,0 +1,49 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://bad.host/
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+      Content-Type:
+      - application/json
+      Authorization:
+      - Basic dXNlcjpwYXNz
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Server:
+      - nginx
+      Date:
+      - Thu, 11 Feb 2021 16:32:25 GMT
+      Content-Type:
+      - text/html; charset=utf-8
+      Transfer-Encoding:
+      - chunked
+      Connection:
+      - close
+      Expires:
+      - Thu, 11 Feb 2021 16:42:25 GMT
+      Cache-Control:
+      - max-age=600
+      X-Frame-Options:
+      - DENY
+    body:
+      encoding: UTF-8
+      string: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html><head><meta
```

Comment:
LOL this has my verizon junk in it.  Wonder if there's a cleaner way to do this.